### PR TITLE
Change folder structure

### PR DESCRIPTION
**Pull request by imvan32**
- [x] Change folder structure
- [x] Move stories
- [x] Move tests
- [x] Change configs
- [x] Run yarn test
- [x] Run yarn format

I have a question about the file.
```
./src/stories/index.d.ts
```
What to do with him?

### CODE REVIEW

**Comment by rot1024** [comment]
> ./src/stories/index.d.ts

This is type definitions for storybooks. How about `src/global.d.ts` ? Thank you

**Comment by rot1024** [comment]
Good job. If your work has been done, please let me know to review the codes.

**Comment by imvan32** [comment]
Ready

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = ({ config, mode }) => ({
         },
       },
       {
-        test: /\.stories\.tsx?$/,
+        test: /(.?stories|story)\.tsx?$/,
```
Comment (rot1024) [inline .storybook/webpack.config.js]: How about `/(\.?stories|story)\.tsx?$/` ?

Comment (imvan32) [reply .storybook/webpack.config.js]: Do you want change this line?

Comment (rot1024) [reply .storybook/webpack.config.js]: Yes, I thought the first dot should be escaped.

Comment (imvan32) [reply .storybook/webpack.config.js]: Ok

**Pull request merged**